src/form-builder/CheckboxFieldGroup.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {
+  useField,
+  FieldHookConfig,
+  FieldArray,
+  useFormikContext,
+} from 'formik';
+
+import {
+  chainArrayValidations,
+  required,
+  requiredLength,
+} from '../utils/validation';
+import CheckboxField from './CheckboxField';
+import { CheckboxGroupProps, CheckboxProps } from './types';
+
+// import CheckboxGroup from 'local-web-components/CheckboxGroup';
+import CheckboxGroup from '@department-of-veterans-affairs/component-library/CheckboxGroup';
+
+const storeInheritedValues = (fieldValue: string[] | undefined) => {
+  if (fieldValue?.length) {
+    const returnKeys = fieldValue.reduce(
+      (
+        newObj: { [key: string]: boolean },
+        current
+      ): { [key: string]: boolean } => {
+        newObj[current] = true;
+        return newObj;
+      },
+      {}
+    );
+    return returnKeys;
+  }
+  return {};
+};
+
+const CheckboxFieldGroup = (props: CheckboxGroupProps): JSX.Element => {
+  // component re-renders these values per form input inside the form
+  const withValidation = {
+    ...props,
+    validate: chainArrayValidations(props, [requiredLength]), // chainValidations(props, [required]),
+  };
+  const [field, meta, helpers] = useField(
+    withValidation as FieldHookConfig<string[]>
+  );
+  // convert the field values to a format that the component stores default values
+  const inheritedValues = storeInheritedValues(field.value);
+
+  return (
+    <>
+      <CheckboxGroup
+        id={props.id}
+        label={props.label}
+        required={!!props.required}
+        name={props.name}
+        values={inheritedValues}
+        options={props.options}
+        onValueChange={(option: CheckboxProps) => {
```

Comment:
Would it be possible to move this onValueChange handler up into the component instead of having it inline here?